Example/ABMSoundCloudAPI/ABMAppDelegate.m

```diff
@@ -8,11 +8,22 @@
 
 #import "ABMAppDelegate.h"
 
+#import "ABMSoundCloudAPISingleton.h"
+
+
+
+
+
 @implementation ABMAppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-    // Override point for customization after application launch.
+	[[ABMSoundCloudAPISingleton sharedManager]setClientID:@"deb3f736ae605f99850f2cc71e9c98b8" secretKey:@"18177564efaef040303d189fa8355bfa"];
+//	[SCSoundCloud setClientID:kVWIConstants_SoundCloudClientId
```

Comment:
Don't comment-out code, just remove it. If you need it again you have always the git history ;)